test/snippet/alignment/scoring/nucleotide_scoring_scheme.cpp

```diff
@@ -0,0 +1,49 @@
+#include <range/v3/view/zip.hpp>
+
+#include <seqan3/alignment/scoring/nucleotide_scoring_scheme.hpp>
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/alphabet/nucleotide/rna15.hpp>
+
+using namespace seqan3;
+
+int main()
+{
+
+{
+//! [two letters]
+nucleotide_scoring_scheme scheme; // hamming is default
+std::cout << "Score between DNA A and G: " << (int) scheme.get_score(dna5::A, dna5::G) << "\n"; // == -1
+std::cout << "Score between DNA A and A: " << (int) scheme.get_score(dna5::A, dna5::A) << "\n"; // == 0
+
+scheme.set_simple_scheme(match_score{3}, mismatch_score{-2});
+std::cout << "Score between DNA A and RNA G: " << (int) scheme.get_score(dna5::A, rna15::G) << "\n"; // == -2
```

Comment:
Maybe add the alphabet size : `Score between DNA5 A and RNA15 G`?

Reply:
:ok: